eng/scripts/Generate-APIView-CodeFile.ps1

```diff
@@ -11,6 +11,10 @@ if (!(Test-Path -Path $ArtifactPath))
   exit 1
 }
 
+
+npm install "@rushstack/node-core-library@5.9.0"
+npm install "js-tokens@9.0.0"
+npm install "@microsoft/api-extractor-model@7.29.8"
 $apiviewParser = "@azure-tools/ts-genapi@2.0.3"
 Write-Host "Installing $($apiviewParser)"
 npm install $apiviewParser --registry $NpmDevopsFeedRegistry
```

Comment:
If moving dependencies start to cause more issues we should consider using a lock file for this install.